src/EnergyPlus/CMakeLists.txt

```diff
@@ -998,8 +998,10 @@ if(BUILD_PACKAGE)
       install(FILES "${PROJECT_SOURCE_DIR}/release/VersionUpdater-NEWS.txt" DESTINATION "PreProcess/" COMPONENT Auxiliary)
       if(WIN32)
         # on Windows, with Build Package, and also Link With Python, we can also drop in shortcuts to the new auxiliary CLI
-        add_executable(windows_gui_launcher WIN32 WindowsGuiLauncher.cc)
-        target_link_options(windows_gui_launcher PRIVATE /SUBSYSTEM:WINDOWS)
+        set(APP_ICON "${PROJECT_SOURCE_DIR}/release/ep.ico")
+        set(RESOURCE_FILE "${PROJECT_BINARY_DIR}/resource.rc")
+        file(GENERATE OUTPUT ${RESOURCE_FILE} CONTENT "IDI_ICON1 ICON \"${APP_ICON}\"\n")
+        add_executable(windows_gui_launcher WIN32 WindowsGuiLauncher.cc ${RESOURCE_FILE})
```

Comment:
This is an untested attempt to embed the E+ icon into the windows_gui_launcher.exe program, which should get icons showing up on the shortcuts in the install folder as well as on the start menu.  Let's see...I'm only like 67% confident.

Reply:
This worked great!  Icons look good.

One quick and very odd note.  @mjwitte and I both experienced this.  When you open EP-Launch (Python) for the first time on Windows, you get a leaf, or feather, or quill icon showing up in the taskbar (I didn't look close).  I confirmed my code was doing the right thing.  Then noticed that on subsequent openings, the icon was the right E+ icon!?  So weird, and Windows fault not mine.  Maybe.